texdoc.cnf

```diff
@@ -93,6 +93,13 @@ suffix_list = doc, -doc, _doc, .doc, /doc, manual, /manual, -manual, userguide,
 #
 # fuzzy_level = 3
 
+## Online URL
+
+# Here you can configure the URL to prompt the user to open when Texdoc
+# is unable to find any local matches.
+
+online_url = https://texdoc.org/serve/PKG/0
```

Comment:
I think it will be better to comment this out in this distribution configuration file because it is already set to the same default value in `texdoclib-config.tlu`. Putting this redundant setting here will change the debug output and it is not ideal.